src/main/java/interface_adapter/delete_user/DeleteViewModel.java

```diff
@@ -0,0 +1,19 @@
+package interface_adapter.delete_user;
+
+import interface_adapter.ViewModel;
+
+import java.beans.PropertyChangeListener;
+import java.beans.PropertyChangeSupport;
+
+public class DeleteViewModel extends ViewModel {
+    private DeleteState state = new DeleteState();
```

Comment:
Don't think you need this

Reply:
I need the DeleteState to be able to use the property fire change observer